src/main/webapp/script.js

```diff
@@ -19,41 +19,47 @@ function getRecipeInfo() {
   params.append('image', image);
   const request = new Request('/dishAnalysis', {method: "POST", body: params});
   fetch(request).then(response => response.json()).then((recipeListInfoJson) => {
-    sessionStorage.recipeList = JSON.parse(recipeListInfoJson);
-    window.location.href = "/display.html";
+    //set top options to radio buttons
+    document.getElementById("first-option-label").innerText = recipeListInfoJson[0];
+    document.getElementById("second-option-label").innerText = recipeListInfoJson[1];
```

Comment:
will there be index out of range if the list size is less than 1 or 2?

Reply:
Yes